### PR TITLE
Unifying listed conditions

### PR DESCRIPTION
**Pull request by daniel-montalvo**
Following up on work from #1506 , these are updates to some rules that are not currently following listing conditions that we agreed on #1275 

---

Closes issue(s):

N/A

Need for Final Call:
This will require a 1 week Final Call << small changes affecting a small number of test cases, if in doubt do not use this. >>

---

### **When creating PR:**

- [x] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [x] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [x] Add yourself (and co-authors) as "Assignees" for PR.
- [x] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [x] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [x] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/audio-as-media-alternative-afb423.md
Comment (WilcoFiers) [inline _rules/audio-as-media-alternative-afb423.md:21]:
```suggestion
This rule applies to every [non-streaming](#non-streaming-media-element) `audio` element for which one of the following are true:
```

File: _rules/audio-text-alternative-e7aa44.md
Comment (WilcoFiers) [inline _rules/audio-text-alternative-e7aa44.md:31]:
```suggestion
This rule applies to any [non-streaming](#non-streaming-media-element) `audio` element for which one of the following are true:
```

File: _rules/audio-transcript-2eb176.md
Comment (WilcoFiers) [inline _rules/audio-transcript-2eb176.md:21]:
```suggestion
This rule applies to every [non-streaming](#non-streaming-media-element) `audio` element for which one of the following are true:
```

File: _rules/form-field-non-empty-accessible-name-e086e5.md
Comment (WilcoFiers) [inline _rules/form-field-non-empty-accessible-name-e086e5.md:25]: I don't think this should be written out in a list like this. I think we've more recently set a president that in cases where we have a bunch of rules, we put the list in the background, and have the applicability say what the conditions are for the roles. This rule has it currently flipped the wrong way around.

File: _rules/html-page-lang-valid-bf051a.md
Comment (WilcoFiers) [inline _rules/html-page-lang-valid-bf051a.md:29]:
```suggestion
This rule applies to any [document element](https://dom.spec.whatwg.org/#document-element) if it is an `html` element for which all of the following is true:
```

File: _rules/html-page-lang-xml-lang-match-5b7ae0.md
Comment (WilcoFiers) [inline _rules/html-page-lang-xml-lang-match-5b7ae0.md:24]:
```suggestion
This rule applies to any [document element](https://dom.spec.whatwg.org/#document-element) if it is an `html` element for which all of the following is true:
```

**Review by WilcoFiers — CHANGES_REQUESTED** [review]
You got a couple of 'm mixed up. Also, can't approve this until the applicability phrasing changes are no longer part of this PR. You can wait for #1506 to get merged, but you might have a heck of a lot of merge conflicts on your hands after that PR merges.

**Comment by daniel-montalvo** [comment]
Closing as it will create a lot of conflicts. Will open a separate one for the remaining rules.

**Pull request closed without merging**
